tests/specs/scss/functions.scss

```diff
@@ -1,29 +1,73 @@
 
-@include test-module('Functions') {
+@include test-module("Functions") {
 
   // Testing set functions
-  @include test('config-set [function]') {
-    @include config-set('test.bar', 'foo');
-    @include config-set('test.color', #fff);
+  @include test("config-set [function] Set a single key.") {
+    @include config-set("bar", "foo");
+    @include config-set("color", #fff);
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.